packages/libs/client/.npmignore

```diff
@@ -1,34 +1,13 @@
-# THIS IS A STANDARD TEMPLATE FOR .npmignore FILES IN THIS REPO.
-
-# Ignore all files by default, to avoid accidentally publishing unintended files.
-*
-
-# Use negative patterns to bring back the specific things we want to publish.
-!/bin/**
-!/lib/**
-!/lib-*/**
-!/dist/**
-!ThirdPartyNotice.txt
-
-# Ignore certain patterns that should not get published.
-/dist/*.stats.*
-/lib/**/test/
-/lib-*/**/test/
-*.test.js
-
-# NOTE: These don't need to be specified, because NPM includes them automatically.
-#
-# package.json
-# README (and its variants)
-# CHANGELOG (and its variants)
-# LICENSE / LICENCE
-
-#--------------------------------------------
-# DO NOT MODIFY THE TEMPLATE ABOVE THIS LINE
-#--------------------------------------------
-
```

Comment:
I guess we need to put some of this back

Reply:
It depends on the chosen philosophy for removing unnecessary files (i.e. whitelist or blacklist).

**For this implementation nothing from the template is required.**

I noticed the few existing `.npmignore` files after finalising my blacklist approach — which I feel is more natural. A whitelist approach is more explicit and is likely to lead to future confusion about why certain files aren't published. The documentation in this template describes how certain files such as "CHANGELOG (and its variants)" are automatically included, however I found this to be incorrect when testing.